cypher/cypher-docs/src/docs/dev/ql/schema-index/index.asciidoc

```diff
@@ -8,24 +8,27 @@ This section explains how to work with indexes in Neo4j and Cypher.
 
 * <<query-schema-index-introduction, Introduction>>
 * <<schema-index-create-a-single-property-index, Create a single-property index>>
+* <<schema-index-create-a-composite-index, Create a composite index>>
 * <<schema-index-get-a-list-of-all-indexes-in-the-database, Get a list of all indexes in the database>>
-* <<create-a-composite-index, Create a composite index>>
 * <<schema-index-drop-a-single-property-index, Drop a single-property index>>
-* <<drop-a-composite-index, Drop a composite index>>
+* <<schema-index-drop-a-composite-index, Drop a composite index>>
 * <<schema-index-use-index, Use index>>
-* <<schema-index-use-a-single-property-index-with-where-using-equality, Use a single-property index with `WHERE` using equality>>
-* <<use-a-composite-index-with-where-using-equality, Use a composite index with `WHERE` using equality>>
-* <<schema-index-use-index-with-where-using-range-comparisons, Use index with `WHERE` using range comparisons>>
-* <<schema-index-use-index-with-in, Use index with `IN`>>
-* <<schema-index-use-index-with-starts-with, Use index with `STARTS WITH`>>
-* <<schema-index-use-index-when-checking-for-the-existence-of-a-property, Use index when checking for the existence of a property>>
+* <<schema-index-equality-check-using-where-single-property-index, Equality check using `WHERE` (single-property index)>>
+* <<schema-index-equality-check-using-where-composite-index, Equality check using `WHERE` (composite index)>>
+* <<schema-index-range-comparisons-using-where-single-property-index, Range comparisons using `WHERE` (single-property index)>>
+* <<schema-index-list-membership-check-using-in-single-property-index, List membership check using `IN` (single-property index)>>
+* <<schema-index-list-membership-check-using-in-composite-index, List membership check using `IN` (composite index)>>
+* <<schema-index-prefix-search-using-starts-with-single-property-index, Prefix search using `STARTS WITH` (single-property index)>>
+* <<schema-index-suffix-search-using-ends-with-single-property-index, Suffix search using `ENDS WITH` (single-property index)>>
+* <<schema-index-substring-search-using-contains-single-property-index, Substring search using `CONTAINS` (single-property index)>>
+* <<schema-index-existence-check-using-exists-single-property-index, Existence check using `exists` (single-property index)>>
 * <<explicit-indexes-procedures, Use built-in procedures to manage and use explicit indexes>>
 
 
 [[query-schema-index-introduction]]
 == Introduction
 
-A database index is a redundant copy of information in the database for the purpose of making retrieving said data more efficient.
+A database index is a redundant copy of the data in the database for the purpose of making more efficient the retrieval of the aforementioned data.
```

Comment:
The phrase `the data` seems to imply `all the data`, while the previous term `information` did not imply `all the information`. If we want to make this change, perhaps clarify with `some of the data`.

Reply:
Also, it is not necessarily the retrieval of the redundant data copy. For example, we make a property redundant in an index in order to find nodes. Perhaps we should say _for the purpose of making searches of related data more efficient_.

Reply:
👍